packages/fluxible-plugin-devtools/src/lib/devtools-plugin.js

```diff
@@ -144,6 +144,10 @@ export default function devToolsPlugin() {
                         return action(ctx, payload, cb);
                     }
 
+                    function timedActionNoCallback(ctx, payload) {
+                        return timedAction(ctx, payload);
```

Comment:
You might want to call these two lines before returning `timedAction`

```js
actionReference.startTime = Date.now();
actionReference.payload = JSON.stringify(payload);
```

Without a start time these actions might not end up in the correct place in the timeline. The payload will also be useful for debugging actions called w/o a callback.